UnitedIncome/components
# [APP-3020] Car, Home and Painting Circle Illustrations

### Description
> Please provide a brief description of your changes below.

Implements the following components.

* `CarCircleIllustration`
* `HomeCircleIllustration`
* `PaintingCircleIllustration`

![car](https://user-images.githubusercontent.com/10888441/60130750-de628380-9765-11e9-8143-c241f7e3e08a.gif)

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* View them all in Storybook, and double check their interaction and story states.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-3020)
